src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,648 @@
+sidecar file, as described in the [BIDS specifications](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#scans-file).
+
+
+### Multiple recordings in a single data file (*_events.tsv)
+
+The `*_events.tsv` should be used to provide information about multiple parts of an acquisition
+session when the data from each of these different recordings is stored in a single data file.
+In such a case, this file is REQUIRED.
+This allows benefiting from the capability of the supported data formats (NIX and NWB) to store multiple
+recordings in a single file, which can be convenient when these recordings share numerous characteristics
+(for example, for subsequent recordings obtained on a single cell in intracellular electrophysiology).
+In such case, the information about these recordings should be stored in columns added in the
+`*_events.tsv` file, which are listed now.
+
+Optional column names in `events.tsv` to support multiple recordings in a single data file:
+
+<!-- TODO: Macro for events -->
+
+
+# BIDS-animal-ephys examples
+
+
+## Examples of toy datasets
+
+
+#### Extracellular Electrophysiology
+
+This dataset contains data from a single subject (subject A), that was recorded on two
+days (2022-01-01 and 2022-01-02).
+On the first day it performed two tasks (nose-poke & rest), and on the second day only a
+rest task was performed.
+Detailed information about these tasks can be found in the `tasks.tsv` and `tasks.json` files.
+The electrophysiology data for each of the three recordings are stored in the corresponding
+session and ephys directories in the `nix` format. Metadata about the probes, their electrodes
+and the corresponding recording channels are stored in `tsv` format. Note that in this case,
+this information is shared between data files (see BIDS Inheritance Principle): in the first session,
+the probe, electrode and channel files apply to both data files of that session, as they do not
+contain a `task` entity in their name. For the nose-poke task, additional behavioral timestamps
+(events) were recorded and stored in an additional `events.tsv` file.
+
+<!-- TODO: Use macro for filetree -->
+
+```
+dataset_description.json
+tasks.tsv
+tasks.json
+participants.tsv
+sub-A/
+  	ses-20220101/
+    		ephys/
+        sub-A_ses-20220101_task-nosepoke_ephys.nix
+        sub-A_ses-20220101_task-nosepoke_ephys.json
+        sub-A_ses-20220101_task-nosepoke_events.tsv
+        sub-A_ses-20220101_task-rest_ephys.nix
+        sub-A_ses-20220101_task-rest_ephys.json
+        sub-A_ses-20220101_channels.tsv
+        sub-A_ses-20220101_electrodes.tsv
+        sub-A_ses-20220101_probes.tsv
+  	ses-20220102/
+    		ephys/
+        sub-A_ses-20220102_task-rest_ephys.nix
+        sub-A_ses-20220102_task-rest_ephys.json
+        sub-A_ses-20220102_channels.tsv
+        sub-A_ses-20220102_electrodes.tsv
+        sub-A_ses-20220102_probes.tsv
+```
```

Comment:
to do: use filetree macro

Reply:
note: we could use a helper for that -- should be trivial to code up in principle and could come handy for others ... let's see if my prompt would produce usable ... chatgpt did quite cool looping trying to make it work but I got claude faster interactively to give a script for which embedded tests passed...

here is the script http://www.oneukrainian.com/tmp/text2tree3.py and for this one I think it produced correct output after I gave it that tab should be just a single space:

```shell
❯ cat ~/.npm/ex | /tmp/text2tree3.py --tab-width 1
{
  "dataset_description.json": "",
  "tasks.tsv": "",
  "tasks.json": "",
  "participants.tsv": "",
  "sub-A/": {
    "ses-20220101/": {
      "ephys/": {
        "sub-A_ses-20220101_task-nosepoke_ephys.nix": "",
        "sub-A_ses-20220101_task-nosepoke_ephys.json": "",
        "sub-A_ses-20220101_task-nosepoke_events.tsv": "",
        "sub-A_ses-20220101_task-rest_ephys.nix": "",
        "sub-A_ses-20220101_task-rest_ephys.json": "",
        "sub-A_ses-20220101_channels.tsv": "",
        "sub-A_ses-20220101_electrodes.tsv": "",
        "sub-A_ses-20220101_probes.tsv": ""
      }
    },
    "ses-20220102/": {
      "ephys/": {
        "sub-A_ses-20220102_task-rest_ephys.nix": "",
        "sub-A_ses-20220102_task-rest_ephys.json": "",
        "sub-A_ses-20220102_channels.tsv": "",
        "sub-A_ses-20220102_electrodes.tsv": "",
        "sub-A_ses-20220102_probes.tsv": ""
      }
    }
  }
}
```

I will adjust all such examples now while at it!

Reply:
FWIW, proposing
- https://github.com/bids-standard/bids-specification/pull/2014